cmd/korg/korg.go

```diff
@@ -46,6 +46,15 @@ Add user to specified orgs:
 Note: Adding to teams is currently unsupported.
 	`
 
+	removeHelpText = `
+Remove users from GitHub orgs and/or teams
+
+Remove user to specified orgs:
+
+	korg remove <github username> --org kubernetes --org kubernetes-sigs
+	korg remove <github username> --org kubernetes --teams kubernetes/kubernetes-maintainers
```

Comment:
Both `korg remove/add ...` throws missing `--teams` flag. Is it a TODO or something to clean?

```
> korg remove Priyankasaggu11929 --org kubernetes --teams kubernetes/kubernetes-maintainers
Error: unknown flag: --teams
Usage:
  korg remove [flags]

```

Reply:
It was a remnant from what I originally planned the command to be.